### PR TITLE
Cleanup files

### PR DESCRIPTION
**Pull request by jiegillet**
Two quick scripts, as discussed in #499 
Please test and review :)

### CODE REVIEW

**Comment by PudottaPommin** [comment]
Looks good! Although I was at 1st confused it's order by name of language and not file extension. Which makes sense, but I would expect order by file extension.

**Comment by jiegillet** [comment]
I don't think extension would make sense, because they are not really extensions, they are arbitrary. For example, we have
```
{
          "lang": "LabVIEW", 
          "name": "LabVIEW"
        },
```
I know that it would be better to stick to extensions in general, but my point holds. 
Language names, on the other hand, are meaningful. 
I'm also sorting `.editorconfig` by language name.

**Comment by PudottaPommin** [comment]
I understand both points. 

One thing. Intention is to run this script from cding into tools/cleanup folder or from root folder of repo.

**Comment by jiegillet** [comment]
Right now you have to be in tools/cleanup and run it from there. 
I'm not actually super happy with this, do you have a suggestion for a better choice? I can't use an absolute path.

**Comment by PudottaPommin** [comment]
Maybe pass folder in which perform cleanup in parameter? Or remove `../../` from paths and considering running it from root folder.

**Comment by jiegillet** [comment]
Parameter is better. No way to mess that up, thank you!

File: tools/cleanup/clean_book_json.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+"""
+Sorts languages in book.json alphabetically into book.json_.
+Check the file manually, then rename.
+You need to path the path to the file as an argument.
```
Comment (Butt4cak3) [inline tools/cleanup/clean_book_json.py]: "pass"?

Comment (jiegillet) [reply tools/cleanup/clean_book_json.py]: Gaaaaaah! 
Thank you -_-

**Comment by jiegillet** [comment]
Muhahaha! Seems I could merge it myself :D

**Pull request merged**